src/frontend/test_runner/tests/testdata/subquery_expr_correlated.yaml

```diff
@@ -211,3 +211,50 @@
     create table t2(x int, y int);
     select x from t1 where y in (select y, x from t2 where t1.x > t2.x + 1000);
   binder_error: 'Bind error: Subquery must return only one column'
+- sql: |
+    create table t1(x int, y int);
```

Comment:
Leave some comments about the scenario each case is testing. For example:
* correlated outer subquery with an uncorrelated inner subquery
* correlated inner subquery with depth = 2
* uncorrelated outer subquery with a correlated inner subquery